includes/admin/payments/class-refund-items-table.php

```diff
@@ -432,7 +433,10 @@ public function display_rows() {
 			</td>
 		</tr>
 
-		<?php if ( edd_use_taxes() ) : ?>
+		<?php
+		$order = $this->get_order();
+		if ( $order && $order->get_tax_rate() ) :
+			?>
```

Comment:
```suggestion
		?>
```

Minor indenting issue.